scipy/sparse/linalg/eigen/arpack/tests/test_arpack.py

```diff
@@ -910,5 +910,20 @@ def test_regression_arpackng_1315():
                         rtol=1e-4)
 
 
+def test_eigs_for_k_greater():
+    # Test eigs() for k beyond limits.
+    A = diags([1, -2, 1], [-1, 0, 1], shape=(4, 4))
+    eigs(A, k=3)
+    eigs(A, k=4)
+    eigs(A, k=5)
+
+
+def test_eigsh_for_k_greater():
+    # Test eigsh() for k beyond limits.
+    A = diags([1, -2, 1], [-1, 0, 1], shape=(4, 4))
+    eigsh(A, k=4)
+    eigsh(A, k=5)
+
+
```

Comment:
Each of these tests should check the returned values of `eigs`/`eigsh` for correctness.